include/dt-bindings/sensor/npcx_tach.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2021 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_SENSOR_NPCX_TACH_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_SENSOR_NPCX_TACH_H_
+
+/* NPCX tachometer port type */
+#define NPCX_TACH_PORT_A     0
+#define NPCX_TACH_PORT_B     1
+
```

Comment:
Add a define for the LFCLK frequency for use in the board device trees.

Reply:
Done.